frontend/src/containers/RecipePage/Recipe.js

```diff
@@ -0,0 +1,134 @@
+      if (difficulty === 'EASY') {
+        return (
+          <img className="recipe-difficulty" src="https://img.icons8.com/material-rounded/24/4EC8ED/low-connection.png" alt="Easy Difficulty Rating" />
+        );
+      }
+      if (difficulty === 'MEDIUM') {
+        return (
+          <img className="recipe-difficulty" src="https://img.icons8.com/material-rounded/24/4EC8ED/medium-connection.png" alt="Medium Difficulty Rating" />
+        );
+      }
+      if (difficulty === 'ADVANCED') {
+        return (
+          <img className="recipe-difficulty" src="https://img.icons8.com/material-rounded/24/4EC8ED/bar-chart.png" alt="Advanced Difficulty Rating" />
+        );
+      }
+      return '';
+    };
+    return (
+      <div className="recipe-overall-container">
+        <div className="row">
+          <div className="header">
+            <Header />
+          </div>
+        </div>
+        <div>
+          <button className="btn navbar-btn btn-lg editButton" type="button">
+            <img src="https://img.icons8.com/windows/32/ffa616/edit.png" alt="edit" />
+            <p id="edit-icon-text">Edit</p>
+          </button>
+          <button className="btn navbar-btn btn-lg cancelButton" type="button">
+            <img src="https://img.icons8.com/windows/32/ffa616/cancel.png" alt="delete" />
+            <p id="delete-icon-text">Delete</p>
+          </button>
+        </div>
+        <img src={recipephoto} alt="recipephoto" className="recipe-photo" />
+        <div className="recipe-specs-bars">
+          <div className="row">
+            <div id="recipe-spec-container1" className="recipe-spec-element col-sm-4 text-center">
+              <span>
+                {mealDifficulty()}
+                {difficulty}
+              </span>
+            </div>
+            <div id="recipe-spec-container2" className="recipe-spec-element col-sm-4 text-center">
+              <span>
+                <i className="fas fa-clock" />
+                { duration }
+                {' '}
+                MIN
+              </span>
+            </div>
+            <div id="recipe-spec-container3" className="recipe-spec-element col-sm-4 text-center">
+              <i className="fas fa-utensil-spoon" />
+              <span>{ servings }</span>
+            </div>
+          </div>
+        </div>
+        <div className="tags-bar">
+          <TagsBar />
+        </div>
+        <div className="row" id="blurb-title">
+          <h1 id="recipe-title" className="col-sm-6 offset-sm-3">
+            {title}
+          </h1>
+          <p id="recipe-descrip" className="col-sm-8 offset-sm-2">
+            {snippet}
+          </p>
+        </div>
+        <div id="hr" />
+        <div className="separator col-sm-10 offset-sm-1" />
+        <div id="ingredient-box">
+          <div id="ingredient-list">
+            <h1 id="ingredient-title">Ingredients</h1>
+            <ul id="ingredient-ul">
+              {ingredient.map(ingred => (
```

Comment:
...we don't need to abbreviate to `ingred` and can instead do `ingredients.map(ingredient => ...`